requirements.txt

```diff
@@ -15,5 +17,8 @@ django-filter  # Filtering support
 #LTI 
 PyLTI1p3==1.9.1
 
+# Not in pypi https://github.com/Harvard-University-iCommons/django-canvas-oauth
+https://github.com/Harvard-University-iCommons/django-canvas-oauth/archive/v1.1.0.tar.gz
```

Comment:
Have you had any communication with these folks? Any plans to keep this updated?

Reply:
No I haven't communicated.

I feel like we'd probably fork their project and extend it if we ever had to do anything with it in the future. Seems better than starting from scratch. The license is fine for that.

Reply:
Okie doke